contributors/THIRD-PARTY-EXTENSION.md

```diff
@@ -0,0 +1,104 @@
+## Introduction
+
+Welcome to the guide for developing external extensions in the `create-eth` repository. This document outlines the workflow for creating and testing your extensions.
+
+## Workflow Overview
+
+### Phase 1: Initial Setup and Development
```

Comment:
More context:

like: "In order to develop an test the extensions locally..... you can follow this workflow...." (just thinking out loud, this shouldn't be the actual text haha)